src/Akkling/Props.fs

```diff
@@ -110,7 +111,7 @@ type Props<'Message> =
           Deploy = Some props.Deploy
           Dispatcher = if props.Dispatcher = Deploy.NoDispatcherGiven then None else Some props.Dispatcher
           Mailbox = if props.Mailbox = Deploy.NoMailboxGiven then None else Some props.Mailbox
-          Router =  if props.RouterConfig = Akka.Routing.RouterConfig.NoRouter then None else Some props.RouterConfig
+          Router =  if props.RouterConfig.Equals NoRouter.Instance then None else Some props.RouterConfig
```

Comment:
`RouterConfig.NoRouter` is deprecated and `props.RouterConfig = NoRouter.Instance` yielded a warning, so I use `Equals`.